pyproject.toml

```diff
@@ -34,6 +34,7 @@ pyright = "^0.0.12"
 pre-commit = "^2.15.0"
 pytest-cov = "^3.0.0"
 setuptools-rust = "^1.1.2"
+pluggy = "1.0.0"
```

Comment:
Is this a leftover from the out-of-tree handlers branch ?

Reply:
Nope, check the commit message:
>  DO NOT MERGE: test poetry.lock consitency checker

Reply:
Aaaah I see :)